source/usage-examples/count.txt

```diff
@@ -0,0 +1,61 @@
+=============================================
+Count the Number of Documents in a Collection
+=============================================
+
+.. default-domain:: mongodb
+
+Overview
+--------
+
+The Node.js driver provides two methods for counting the number of
+documents in a collection:
+
+- :node-api:`collection.countDocuments()
+  </Collection.html#countDocuments>` returns the number of documents in
+  the collection that match the specified query. If you specify a empty
+  query document, ``countDocuments()`` returns the total number of
+  documents in the collection.
+
+- :node-api:`collection.estimatedDocumentCount
```

Comment:
Same as above comment, just making sure you're aware

Reply:
Ack.